docker/api/Dockerfile

```diff
@@ -12,27 +12,30 @@ LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security
 
 ARG CISA_UID=421
 ENV CISA_HOME="/home/cisa"
-ENV ECHO_MESSAGE="Hello World from Dockerfile"
 
 RUN addgroup --system --gid ${CISA_UID} cisa \
   && adduser --system --uid ${CISA_UID} --ingroup cisa cisa
 
-RUN apk --update --no-cache add \
-ca-certificates \
-openssl \
-py-pip
+# RUN apk --update --no-cache add \
+
+RUN apt-get update && \
+ apt-get install wget
+
+# ca-certificates \
+# openssl \
+# py-pip
```

Comment:
Why are these packages removed and not just updated to reflect the Debian packages?

Reply:
We just weren't sure if they should be included and scaled back dependencies in the build to isolate errors in testing. We are totally fine with adding them back in if that's the preference.